UPGRADE-4.2.md

```diff
@@ -180,6 +183,22 @@ FrameworkBundle
  * Deprecated support for legacy translations directories `src/Resources/translations/` and `src/Resources/<BundleName>/translations/`, use `translations/` instead.
  * Support for the legacy directory structure in `translation:update` and `debug:translation` commands has been deprecated.
 
+HttpFoundation
+--------------
+
+ * The default value of the "$secure" and "$samesite" arguments of Cookie's constructor
+   will respectively change from "false" to "null" and from "null" to "lax" in Symfony
+   5.0, you should define their values explicitly or use "Cookie::create()" instead.
```

Comment:
quotes should be replaced by backticks btw

Reply:
Nice catch